dali/python/nvidia/dali/_debug_mode.py

```diff
@@ -131,6 +131,10 @@ def __init__(self, exec_func, **kwargs):
         self._cur_subpipeline_id = -1
         self._exec_func = exec_func
 
+        import numpy as np
+        seed = kwargs.get('seed', np.random.randint(0, 2**32))
```

Comment:
Just a nitpick: how it would work if someone passes `-1` (as it's the default value that indicates random seed in regular pipeline).

I'm wondering if we maybe should repeat the regular pipeline constructor keyword args in this constructor as well - but that would be a separate PR probably. Let's discuss.

Reply:
done, generating new seed when seed < 0 (same as in regular Pipeline)